testdata/baselines/reference/submoduleAccepted/compiler/commonjsAccessExports.types.diff

```diff
@@ -1,37 +1,16 @@
 --- old.commonjsAccessExports.types
 +++ new.commonjsAccessExports.types
-@@= skipped -2, +2 lines =@@
- === /a.js ===
- exports.x = 0;
- >exports.x = 0 : 0
-->exports.x : 0
-->exports : typeof import("/a")
-->x : 0
-+>exports.x : any
-+>exports : any
-+>x : any
- >0 : 0
- 
- exports.x;
-->exports.x : 0
-->exports : typeof import("/a")
-->x : 0
-+>exports.x : any
-+>exports : any
-+>x : any
- 
- // Works nested
+@@= skipped -16, +16 lines =@@
  {
      // 'exports' does not provide a contextual type to a function-class
      exports.Cls = function() {
 ->exports.Cls = function() {        this.x = 0;    } : typeof Cls
 ->exports.Cls : typeof Cls
-->exports : typeof import("/a")
-->Cls : typeof Cls
-->function() {        this.x = 0;    } : typeof Cls
 +>exports.Cls = function() {        this.x = 0;    } : () => void
 +>exports.Cls : any
```

Comment:
I'm not sure why `exports.Cls : any` but it's not a problem with `exports` resolution, because `exports.x` works above. I suspect it's something wrong with partially implemented construction function typing.